examples/InformationPanel.horizontal.tsx

```diff
@@ -0,0 +1,85 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import {
+  Button,
+  InformationPanel,
+  InformationPanelBody,
+  InformationPanelContent,
+  InformationPanelHeader,
+  InformationPanelWrapper,
+  Input,
+  Label,
+  Table,
+  Text,
+  InputGrid,
+} from '@itwin/itwinui-react';
+import * as React from 'react';
+
+export default () => {
+  const [openRowIndex, setOpenRowIndex] = React.useState<number>(1);
+
+  return (
+    <InformationPanelWrapper>
+      <Table
+        columns={[
+          { id: 'name', Header: 'Name', accessor: 'name' },
+          {
+            Header: 'Details',
+            Cell: ({ row }) => (
+              <Button onClick={() => setOpenRowIndex(row.index)}>
+                Details
+              </Button>
+            ),
+          },
+        ]}
+        data={[...Array(3).fill(null)].map((_, index) => ({
+          name: `Row${index}`,
+        }))}
+        emptyTableContent='No data.'
+        style={{ minWidth: '450px' }}
+      />
+
+      <InformationPanel
+        isOpen={openRowIndex != undefined && openRowIndex !== -1}
+        orientation='horizontal'
+      >
+        <InformationPanelHeader onClose={() => setOpenRowIndex(-1)}>
+          <Text variant='subheading'>Row {openRowIndex ?? 0}</Text>
+        </InformationPanelHeader>
+        <InformationPanelBody>
+          <InformationPanelContent displayStyle='inline'>
```

Comment:
Should one example show the inline `InformationPanelContent` and the other show the default?

Reply:
i don't think it's necessary, `displayStyle='inline'` is the main reason to use this component, otherwise it's just `<Flex flexDirection='column'>`

Reply:
That's true. Then I guess it would at least be nice to show some variation in the two informational panel examples. Because currently, both panels look nearly identical. Maybe the main example can look similar to the [InformationPanel -> Basic story](https://itwin.github.io/iTwinUI/react/?story=informationpanel--basic&theme=dark) with just one column?

<img width="386" alt="Screenshot 2023-12-19 at 3 25 52 PM" src="https://github.com/iTwin/iTwinUI/assets/45748283/1b1b30c7-c5ec-416d-b6ae-2cc3da1d5141">

Reply:
how about i remove InformationPanelContent from the main example? it feels too specific, and is not required for using InformationPanel.

Reply:
Yes, that sounds better. Because if both examples use InformationPanelContent, it may give the impression that InformationPanelContent is a required wrapper around the content.

Reply:
removed from main example and simplified